Mage/src/test/java/mage/ManaTest.java

```diff
@@ -145,7 +145,7 @@ public void shouldNotCreateManaFromNullColoredManaSymbol() {
     public void shouldCreateManaFromIntegers() {
 
         // when
-        Mana mana = new Mana(1, 2, 3, 4, 5, 6, 7, 8);
+        Mana mana = new Mana(4, 3, 5, 1, 2, 6, 7, 8);
```

Comment:
1, 2, 3, 4, 5 for easy test reading/write. So the best way -- change assert color order, not init.

Reply:
yeah I'm expecting to have to change tests around